ukrainian/break-through-wall/try-the-taste/continue-eating/wait/wait.md

```diff
@@ -1,7 +1,7 @@
 Ви чекаєте. І тут раптом чуєте гуркіт, обертаєтесь і бачите, що стіна пробита.
 Через отвір видно  постать гнома. Він запрошує вас зайти до себе.
 
-Приймаєте запрошення гнома і [заходите](go-with-dwarf/go-with-dwarf.md) у діру...
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.